src/controllers/activegate/reconciler/statefulset/reconciler.go

```diff
@@ -117,13 +118,13 @@ func (r *Reconciler) buildDesiredStatefulSet() (*appsv1.StatefulSet, error) {
 		return nil, errors.WithStack(err)
 	}
 
-	cpHash, err := r.calculateCustomPropertyHash()
+	agHash, err := r.calculateActiveGateHash()
```

Comment:
```suggestion
	agHash, err := r.calculateActiveGateConfigurationHash()
```
`ActiveGateHash` may by confusing because one may think it's image hash.